launch/app/runtime/services.cfg

```diff
@@ -6,9 +6,7 @@ folder:persistence=persist
 folder:things=things
 
 basicui:defaultSitemap=demo
-basicui:enableIcons=true
-basicui:condensedLayout=false
-basicui:capitalizeValues=false
+basicui:inlineSvg=false
```

Comment:
Could you briefly elaborate these changes? I am not fully clear, why they are done.

Reply:
Ah, ok, it is the demo setup and you simply removed the things that are anyhow the default values, I guess.

Reply:
The Basic UI settings are now on client side (browser).
They do not exist anymore on OH server side, except inlineSvg and defaultSitemap.